### ISSUE TITLE
Restrict oauth access to organization members

### ISSUE THREAD

**Issue by jonasdkhansen** [kubernetes-helm/monocular#427]
It would be nice, if we could restrict oauth login, to only members of a specific organization, like it's possible with Jenkins and Rancher.

How do you protect access to your UI right now ? Or are your UI just not exposed to the outside world ?

**Comment by prydonius** [comment]
I agree this would be useful in the GitHub oauth integration. @jonasdkhansen in the meantime, you could wrap the UI around https://github.com/bitly/oauth2_proxy which has many options to restrict access.
